indico/htdocs/js/indico/modules/cephalopod.js

```diff
@@ -71,7 +71,7 @@ function initCephalopdOnSettingsPage(cephalopodUrl) {
     if (!enabled) {
         defer.resolve({
             'title'   : $T('Disabled'),
-            'subtitle': $T('Your are not part of the community.'),
+            'subtitle': $T('You are not registered as part of the community.'),
```

Comment:
I guess you mean "Your instance is not **registered** as part of the community"?
I'd rather say "This server is not registered as part of the community", as "instance" may seem a bit without a context.